lib/beaker/host_prebuilt_steps.rb

```diff
@@ -505,10 +522,13 @@ def set_env host, opts
         host.exec(Command.new("touch #{host[:ssh_env_file]}"))
 
         #add the constructed env vars to this host
-        host.add_env_var('RUBYLIB', '$RUBYLIB')
-        host.add_env_var('PATH', '$PATH')
-        env.each_pair do |var, value|
-          host.add_env_var(var, value)
+
+        if host['platform'] !~ /freebsd/
```

Comment:
Did you try with both FOSS/PE puppet?  It is weird that the pathing would _just_ work.